dns_config/test/api_acl_test.go

```diff
@@ -7,87 +7,158 @@ Testing AclAPIService
 
 // Code generated by OpenAPI Generator (https://openapi-generator.tech);
 
-package dns_config
+package dns_config_test
 
 import (
+	"bytes"
 	"context"
+	"encoding/json"
+	"io"
+	"net/http"
 	"testing"
 
-	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
 	openapiclient "github.com/infobloxopen/bloxone-go-client/dns_config"
 	"github.com/infobloxopen/bloxone-go-client/internal"
 )
 
-func Test_dns_config_AclAPIService(t *testing.T) {
-
-	configuration := internal.NewConfiguration()
-	apiClient := openapiclient.NewAPIClient(configuration)
-
-	t.Run("Test AclAPIService AclCreate", func(t *testing.T) {
+var dns_config_Post = openapiclient.ConfigACL{
+	Comment: openapiclient.PtrString("This is a dummy ACL for testing."),
+	Id:      openapiclient.PtrString("dummyAclId"),
+	Name:    "dummyAclName",
+	Tags:    make(map[string]interface{}),
+}
 
-		t.Skip("skip test") // remove to run test
+var dns_config_Patch = openapiclient.ConfigACL{
+	Comment: openapiclient.PtrString("This is an updated dummy ACL for testing."),
+	Id:      dns_config_Post.Id,
+	Name:    "updatedDummyAclName",
+	Tags:    make(map[string]interface{}),
+}
 
-		resp, httpRes, err := apiClient.AclAPI.AclCreate(context.Background()).Execute()
+func Test_dns_config_AclAPIService(t *testing.T) {
```

Comment:
You can remove the package name from the tests

```suggestion
func TestAclAPIService(t *testing.T) {
```